tanner/emulators/php_code_injection.py

```diff
@@ -0,0 +1,36 @@
+import aiohttp
+import asyncio
+import logging
+
+from tanner.utils import patterns
+
+class PHPCodeInjection:
+    def __init__(self, loop=None):
+        self._loop = loop if loop is not None else asyncio.get_event_loop()
+        self.logger = logging.getLogger('tanner.php_code_injecton')
+
+    async def get_injection_result(self, code): 
+        code_injection_result = None
+        code = '<?php eval(\'$a = {code}\'); ?>'.format(code=code)
+        try:
+            async with aiohttp.ClientSession(loop=self._loop) as session:
+                async with session.post('http://127.0.0.1:8088/', data=code) as resp:
+                    code_injection_result = await resp.json()
+        except aiohttp.ClientError as client_error:
+            self.logger.error('Error during connection to php sandbox %s', client_error)
+        else:
+            await resp.release()
```

Comment:
http://aiohttp.readthedocs.io/en/stable/client_reference.html#aiohttp.ClientResponse.release

and also

https://github.com/aio-libs/aiohttp/blob/44edff8a3a930e4ecc35773a48c300c29b9919ae/aiohttp/client.py#L608

So I think we don't need to call release :)
with statement makes it for us, I suppose